SPECS/glassfish-servlet-api/glassfish-servlet-api.spec

```diff
@@ -21,7 +21,7 @@ Distribution:   Mariner
 %global artifactId javax.servlet-api
 Name:           glassfish-servlet-api
 Version:        3.1.0
```

Comment:
The latest version is 4.0.1.
https://github.com/javaee/servlet-spec/commit/5574e9b39d806120ac4d3539826a75b2cea0165a